src/components/forms/account/AccountForm.tsx

```diff
@@ -201,21 +207,7 @@ export default function AccountForm({ onSave }: AccountFormProps): JSX.Element {
 async function onSubmit(data: FormValues, onSave: Function) {
   const account = await Account.create({ ...data }).save();
 
-  mutate(
-    '/api/accounts',
-    async (accounts: AccountsMap) => {
-      const [child, parent] = await Promise.all([
-        Account.findOneByOrFail({ guid: account.guid }),
-        Account.findOneByOrFail({ guid: account.parent.guid }),
-      ]);
-      return {
-        ...accounts,
-        [child.guid]: child,
-        [parent.guid]: parent,
-      };
-    },
-    { revalidate: false },
-  );
```

Comment:
I added this code because I thought there were performance problems loading things from SWR but it was apexcharts re-rendering. I'm deleting this code now as it works fast with chart.js plus, it has an edge case for when the parent is a root placeholder (i.e. topmost assets, expenses, etc) where we should be updating `type_asset` too in here. It can be done but adds a bunch of complexity and this brings no gains right now